subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/generator/JavaTestProject.groovy

```diff
@@ -41,8 +41,8 @@ enum JavaTestProject {
     LARGE_MONOLITHIC_GROOVY_PROJECT(new TestProjectGeneratorConfigurationBuilder("largeMonolithicGroovyProject", Language.GROOVY)
         .withSourceFiles(50000)
         .withSubProjects(0)
-        .withDaemonMemory('2g')
-        .withCompilerMemory('4g')
+        .withDaemonMemory('3g')
+        .withCompilerMemory('6g')
```

Comment:
Is the problem the daemon memory or the compiler memory? My guess would be that the project doesn't need so much daemon memory....